src/genie/libs/parser/iosxe/tests/ShowBootflash/cli/equal/golden_output_1_expected.py

```diff
@@ -0,0 +1,4 @@
+expected_output = {
+    "available": "13755338752",
+    "used": "489017344"
+}
```

Comment:
let's capture all the output like. and `bytes_` added to `available` and `used` so that we can notice the unit from key name.
```suggestion
expected_output = {
    "bytes_available": "13755338752",
    "bytes_used": "489017344"
    "index": {
    	1: {
    		"length": 0,
    		"date_time": "Oct 12 2020 07:23:32 +00:00",
    		"path": /bootflash/
    	},
    	2: {
(snip)
```

Reply:
confirmed the change